includes/Pages/PageMain.php

```diff
@@ -68,22 +68,29 @@ protected function main()
 
     /**
      * @param PdoDatabase $database
+     * @param SiteConfiguration $config
      * @param bool        $seeAllRequests
      *
      * @internal param User $currentUser
      */
-    private function setupLastFiveClosedData(PdoDatabase $database, $seeAllRequests)
+    private function setupLastFiveClosedData(PdoDatabase $database, SiteConfiguration $config, $seeAllRequests)
```

Comment:
Given this is a private function and thus completely inaccessible to anything outside the class (eg, unit tests, or other classes which may (actually don't) inherit from this class, *and* that the `$config` from the caller of the function is actually sourced from `$this->getSiteConfiguration()`, is it necessary to add this as a parameter instead of just pulling the config object directly in the function?